hack/remove-members.go

```diff
@@ -0,0 +1,179 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"sort"
+	"strings"
+)
+
+var dryrun bool
+var configPath string
+
+func main() {
+	flag.StringVar(&configPath, "path", ".", "Path to config directory/subdirectory")
+	flag.BoolVar(&dryrun, "dryrun", true, "Enable Dryrun or not. This flag controls whether the changes are simulated or live. If the changes are simulated it will only print the removal details and the commit message")
+
+	flag.Parse()
+
+	flag.Usage = func() {
+		fmt.Fprintf(os.Stderr, "Usage: remove-members [flags] <file-containing-members-list>\n")
```

Comment:
would be good to list out the flags here. http://courses.cms.caltech.edu/cs11/material/general/usage.html has some details on the format

Reply:
`flag.PrintDefaults()` prints the details of the flag. The current output is 
```
Usage: remove-members [flags] <file-containing-members-list>
  -dryrun
        Enable Dryrun or not. This flag controls whether the changes are simulated or live. If the changes are simulated it will only print the removal details and the commit message (default true)
  -path string
        Path to config directory/subdirectory (default ".")
```

Or do you mean to provide the flags in the header line itself?

Reply:
`Usage: remove-members [-dryrun] [-path] member-file (file-containing-members-list)`

Is this fine?